pkg/testutil/testregistry/testregistry_linux.go

```diff
@@ -232,7 +235,78 @@ func (ba *BasicAuth) Params(base *testutil.Base) []string {
 	return ret
 }
 
+func NewIPFSRegistry(base *testutil.Base, ca *testca.CA, port int, auth Auth, boundCleanup func(error)) *RegistryServer {
+	EnsureImages(base)
+
+	name := testutil.Identifier(base.T)
+	// listen on 0.0.0.0 to enable 127.0.0.1
+	listenIP := net.ParseIP("0.0.0.0")
```

Comment:
Why not  127.0.0.1?

Reply:
Then it will not allow access using (non-localhost) host-ip, reducing possible test scope (comment is slightly misleading, originating from previous code in the registry start function).